ui/components/CustomToolbarSelect.js

```diff
@@ -12,143 +12,125 @@ import MesheryChartDialog from './MesheryChartDialog';
 import MesheryChart from './MesheryChart';
 import { clearResultsSelection } from '../lib/store';
 
-const defaultToolbarSelectStyles = { iconButton : {
-  marginRight : '24px',
-  top : '50%',
-  display : 'inline-block',
-  position : 'relative',
-},
-icon : { color : '#000', },
-inverseIcon : { transform : 'rotate(90deg)', }, };
+const defaultToolbarSelectStyles = {
+  iconButton : {
+    marginRight : '24px',
+    top : '50%',
+    display : 'inline-block',
+    position : 'relative',
+  },
+  icon : {
+    color : '#000',
+  },
+  inverseIcon : {
+    transform : 'rotate(90deg)',
+  },
+};
 
-class CustomToolbarSelect extends React.Component {
-//   handleClickInverseSelection = () => {
-//     const nextSelectedRows = this.props.displayData.reduce((nextSelectedRows, _, index) => {
-//       if (!this.props.selectedRows.data.find(selectedRow => selectedRow.index === index)) {
-//         nextSelectedRows.push(index);
-//       }
+function CustomToolbarSelect({ classes, results_selection, setSelectedRows, clearResultsSelection }) {
+  const [dialogOpen, setDialogOpen] = useState(false);
+  const [data, setData] = useState([]);
+  const [, setChartCompare] = useState([]);
```

Comment:
Why do we need this state if don't use it anywhere?

Reply:
We are not using it now but I still have this here because it was there in the class bases component and maybe we can use it for logging and all.